wof/core_1_1.py

```diff
@@ -605,21 +605,17 @@ def create_source_element(self, sourceResult):
 
     def create_contact_info_element(self, sourceResult):
 
-        if (sourceResult.Address and
-           sourceResult.City and
-           sourceResult.State and
-           sourceResult.ZipCode):
+        if sourceResult.ContactName:
 
-            addressString = ", ".join([sourceResult.Address,
-                                       sourceResult.City,
-                                       sourceResult.State,
-                                       sourceResult.ZipCode])
+            addressString = ''
+            if sourceResult.Address:
+                addressString = '{}'.format(sourceResult.Address)
 
             contactInfo = WaterML.ContactInformationType(
-                email=sourceResult.Email,
+                email=[sourceResult.Email],
                 contactName=sourceResult.ContactName,
-                phone=sourceResult.Phone,
-                address=addressString)
+                phone=[sourceResult.Phone],
+                address=[addressString])
```

Comment:
I believe you misunderstood my comment 😄 
My suggestion was basic defensive programming techniques that are not needed when you do your homework, like you did.

Since you **do** know that there won't be list arguments passed here your solution is fine and list of lists will not be created.